bootstrap.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+sudo apt-get update
+
+# See: https://github.com/Toblerity/Fiona/issues/74
+sudo apt-get install -y python-software-properties
+sudo add-apt-repository ppa:ubuntugis/ppa
```

Comment:
This is for a more recent version of GDAL? Does needing this change if we are on 14.04?